krux_sqs/sqs.py

```diff
@@ -192,3 +197,17 @@ def delete_messages(self, queue_name, messages):
             )
         else:
             self._logger.debug('Messages list is empty. Not deleting any messages.')
+
+    def send_message(self, queue_name, message):
+        """
+        Send the given message to the given queue.
+
+        :param queue_name: :py:class:`str` Name of the queue to send messages.
+        :param message: :py:class:`dict` Message dict to send.
+        """
+        # GOTCHA: queue.delete_messages() does not handle an empty list
+        if message:
+            self._logger.debug('Send following message: %s', message)
+            self._get_queue(queue_name).send_message(MessageBody=simplejson.dumps(message))
```

Comment:
Let's provide a way to send raw message as is. What if I just wanted to send a raw string to SQS?